core/validation/initial_validation/schemas.py

```diff
@@ -106,3 +106,30 @@
         '"Future_High_Street_Fund".',
     ),
 ]
+# TODO replace the placeholder error messages with those from design
```

Comment:
are there Jira tickets tracking these TODO items?

Reply:
There is a ticket that Kate is owning at the moment to write up all the messages and that will eventually get passed to us to implement - so yes for this